diff --git a/toxcore/toxav/audio.h b/toxcore/toxav/audio.h
index 358fe4e..3734987 100644
--- a/toxcore/toxav/audio.h
+++ b/toxcore/toxav/audio.h
@@ -5,7 +5,7 @@
 #ifndef C_TOXCORE_TOXAV_AUDIO_H
 #define C_TOXCORE_TOXAV_AUDIO_H
 
-#include <opus.h>
+#include "opus.h"
 #include <pthread.h>
 
 #include "toxav.h"
diff --git a/toxcore/toxav/groupav.h b/toxcore/toxav/groupav.h
index a280153..c74d6da 100644
--- a/toxcore/toxav/groupav.h
+++ b/toxcore/toxav/groupav.h
@@ -6,7 +6,7 @@
 #define C_TOXCORE_TOXAV_GROUPAV_H
 
 // Audio encoding/decoding
-#include <opus.h>
+#include "opus.h"
 
 #include "../toxcore/group.h"
 #include "../toxcore/tox.h"
diff --git a/toxcore/toxcore/bin_pack.h b/toxcore/toxcore/bin_pack.h
index 0ef05b4..25f9d7a 100644
--- a/toxcore/toxcore/bin_pack.h
+++ b/toxcore/toxcore/bin_pack.h
@@ -5,7 +5,7 @@
 #ifndef C_TOXCORE_TOXCORE_BIN_PACK_H
 #define C_TOXCORE_TOXCORE_BIN_PACK_H
 
-#include <msgpack.h>
+#include "msgpack.h"
 #include <stdbool.h>
 #include <stdint.h>
 
diff --git a/toxcore/toxcore/bin_pack.m b/toxcore/toxcore/bin_pack.m
index e937710..69d3808 100644
--- a/toxcore/toxcore/bin_pack.m
+++ b/toxcore/toxcore/bin_pack.m
@@ -4,7 +4,7 @@
 
 #include "bin_pack.h"
 
-#include <msgpack.h>
+#include "msgpack.h"
 
 void bin_pack_array(msgpack_packer *mp, size_t size)
 {
diff --git a/toxcore/toxcore/bin_unpack.h b/toxcore/toxcore/bin_unpack.h
index 3751153..a4a4630 100644
--- a/toxcore/toxcore/bin_unpack.h
+++ b/toxcore/toxcore/bin_unpack.h
@@ -5,7 +5,7 @@
 #ifndef C_TOXCORE_TOXCORE_BIN_UNPACK_H
 #define C_TOXCORE_TOXCORE_BIN_UNPACK_H
 
-#include <msgpack.h>
+#include "msgpack.h"
 #include <stdint.h>
 
 #include "attributes.h"
diff --git a/toxcore/toxcore/bin_unpack.m b/toxcore/toxcore/bin_unpack.m
index ee3cd35..a5bb6bb 100644
--- a/toxcore/toxcore/bin_unpack.m
+++ b/toxcore/toxcore/bin_unpack.m
@@ -4,7 +4,7 @@
 
 #include "bin_unpack.h"
 
-#include <msgpack.h>
+#include "msgpack.h"
 
 #include "ccompat.h"
 
diff --git a/toxcore/toxcore/crypto_core.m b/toxcore/toxcore/crypto_core.m
index ebaa9fa..4f4f83c 100644
--- a/toxcore/toxcore/crypto_core.m
+++ b/toxcore/toxcore/crypto_core.m
@@ -16,7 +16,7 @@
 
 #ifndef VANILLA_NACL
 // We use libsodium by default.
-#include <sodium.h>
+#include "sodium.h"
 #else
 #include <crypto_box.h>
 #include <crypto_hash_sha256.h>
diff --git a/toxcore/toxcore/events/events_alloc.h b/toxcore/toxcore/events/events_alloc.h
index 4f1f3d3..29eaba4 100644
--- a/toxcore/toxcore/events/events_alloc.h
+++ b/toxcore/toxcore/events/events_alloc.h
@@ -5,7 +5,7 @@
 #ifndef C_TOXCORE_TOXCORE_TOX_EVENTS_INTERNAL_H
 #define C_TOXCORE_TOXCORE_TOX_EVENTS_INTERNAL_H
 
-#include <msgpack.h>
+#include "msgpack.h"
 
 #include "../attributes.h"
 #include "../tox_events.h"
diff --git a/toxcore/toxcore/network.m b/toxcore/toxcore/network.m
index f4d820b..c81dd32 100644
--- a/toxcore/toxcore/network.m
+++ b/toxcore/toxcore/network.m
@@ -89,7 +89,7 @@
 
 #ifndef VANILLA_NACL
 // Used for sodium_init()
-#include <sodium.h>
+#include "sodium.h"
 #endif
 
 #include "ccompat.h"
diff --git a/toxcore/toxcore/tox_events.m b/toxcore/toxcore/tox_events.m
index 4de5c0b..93bdc36 100644
--- a/toxcore/toxcore/tox_events.m
+++ b/toxcore/toxcore/tox_events.m
@@ -4,7 +4,7 @@
 
 #include "tox_events.h"
 
-#include <msgpack.h>
+#include "msgpack.h"
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
diff --git a/toxcore/toxcore/tox_unpack.h b/toxcore/toxcore/tox_unpack.h
index d3638b6..687b865 100644
--- a/toxcore/toxcore/tox_unpack.h
+++ b/toxcore/toxcore/tox_unpack.h
@@ -5,7 +5,7 @@
 #ifndef C_TOXCORE_TOXCORE_TOX_UNPACK_H
 #define C_TOXCORE_TOXCORE_TOX_UNPACK_H
 
-#include <msgpack.h>
+#include "msgpack.h"
 
 #include "attributes.h"
 #include "tox.h"
diff --git a/toxcore/toxcore/tox_unpack.m b/toxcore/toxcore/tox_unpack.m
index 2c5a81c..bb5babc 100644
--- a/toxcore/toxcore/tox_unpack.m
+++ b/toxcore/toxcore/tox_unpack.m
@@ -4,7 +4,7 @@
 
 #include "tox_unpack.h"
 
-#include <msgpack.h>
+#include "msgpack.h"
 #include <stdint.h>
 
 #include "bin_unpack.h"
diff --git a/toxcore/toxencryptsave/toxencryptsave.m b/toxcore/toxencryptsave/toxencryptsave.m
index aaa4916..28de4e4 100644
--- a/toxcore/toxencryptsave/toxencryptsave.m
+++ b/toxcore/toxencryptsave/toxencryptsave.m
@@ -8,7 +8,7 @@
  */
 #include "toxencryptsave.h"
 
-#include <sodium.h>
+#include "sodium.h"
 
 #include <stdlib.h>
 #include <string.h>
